### PR TITLE
Update streams.json

### PR DESCRIPTION
**Pull request by kiriakos**
Make sure the commit workflow works

### CODE REVIEW

**Review by safestak-keith — APPROVED** [review]
Looks good! @kiriakos

File: streams.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "blurb": "The Cardano Revolution. Why you should build on Cardano.",
     "url":   "https://www.youtube.com/watch?v=sM0_V53_kGo",
     "language": "en",
-    "timestamp": 1614803442,
```
Comment (asos-keith) [inline streams.json:7]: Testing comments from another account

File: streams.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
+    "timestamp": 1614803400,
```
Comment (safestak-keith) [inline streams.json:7]: Looks fine to me

**Review by safestak-keith — APPROVED** [review]
Completing review

**Review by safestak-keith — APPROVED** [review]

**Review by safestak-keith — APPROVED** [review]

**Review by safestak-keith — APPROVED** [review]
Finishing review

**Pull request merged**